community/modules/scheduler/cloud-batch-login-node/variables.tf

```diff
@@ -70,3 +70,13 @@ variable "batch_job_directory" {
   type        = string
   default     = "/home/batch-jobs"
 }
+
+variable "enable_oslogin" {
+  description = "Enable or Disable OS Login with \"ENABLE\" or \"DISABLE\". Set to \"INHERIT\" to inherit project OS Login setting."
```

Comment:
Is there a reason to not just set this to either "TRUE", "FALSE" or null (which implies "INHERIT")? ENABLE/DISABLE seems to add another layer that's inconsistent with the actual API. 

Alternatively, you could just have a bool that defaults to null, if not set it inherits.

Reply:
I am following the same pattern as [vm-instance](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/199ab9a0a6026b1fb1474bced569f02f7716851f/modules/compute/vm-instance/variables.tf#L211). I might try to provide more context offline.